image/rhel/fetch-stackrox-data.sh

```diff
@@ -16,9 +20,9 @@ fetch_stackrox_data() {
 
     sha256sum -c <( echo "$(cat /tmp/external-networks/checksum) /tmp/external-networks/networks" )
 
-    mkdir /stackrox-data/external-networks
-    zip -jr /stackrox-data/external-networks/external-networks.zip /tmp/external-networks
+    mkdir -p "${target_dir}/external-networks"
+    zip -jr "${target_dir}/external-networks/external-networks.zip" /tmp/external-networks
```

Comment:
I don't know why we have `--test` option in `zip` command downstream and if it ever saved us from anything but it looks like something which wouldn't hurt having.
```suggestion
    zip -jr --test "${target_dir}/external-networks/external-networks.zip" /tmp/external-networks
```